docs/operation_center/intakes.md

```diff
@@ -1,33 +1,61 @@
 
 All features related to intakes are visible through the “Intakes” menu on the Operations Center.
-The intakes homepage will allow you to view the list of intakes already created within the community but also to create intakes.
 
-## Intake List
+# 📄 Intakes Homepage
+
+The intakes homepage allows you to view the list of intakes already created within the community but also to create new intakes.
+
+On this page, you can find: 
+
+- The created intakes in your community
+- The entity associated with the intake
+- The number of events sent to SEKOIA.IO for the past 7 days
+- The number of events in error
+- The intake key for each intake that you can copy directly from the table to your clipboard
+- The creator as well as the creation date
+- A delete button that implies the events produced by the intake will no longer be analyzed.
+
+![intakes-listing.png](../../assets/operation_center/intakes-listing.png)
+
+# 📐 Create an intake using the integrations catalog
+
+To configure a new source of events in your community, you can rely on our list of continuously growing integrations that are constantly developed and enhanced by SEKOIA.IO’s team. 
+
+![SEKOIA.IO’s various sources of events](../../assets/operation_center/intakes-types.png)
 
-On the intake list, the number of events sent to SEKOIA.IO for the past 7 days and the number of events in error (not integrated) are displayed, as well as the intake key, with a button to copy this value to the clipboard.
+SEKOIA.IO’s various sources of events
 
-![image](https://user-images.githubusercontent.com/94441479/158421171-a0f7a1a8-4c9b-4c9f-acae-97a00f6e0fc9.png)
+To create an intake, you have to: 
 
-## Create an intake using the integrations catalog
+1. Click on the “+ Intake” from the Intakes homepage
+2. Choose the source of events that suits your needs (you can filter these sources by clicking on the categories’ tags
+3. Click on the “Create” button in the source card 
+4. Provide an intelligible name
+5. Select the entity to which you want to associate the corresponding data
+6. Click on Create 
 
-After clicking on the dedicated button “+ Intake”, you will be able to create an intake with an existing integration developed by SEKOIA.IO team.
-You just need to choose the right source in the catalog and click on “Create”.
-Here is the information you should provide to validate the creation:
+![modal-intake-creation.png](../../assets/operation_center/modal-intake-creation.png)
 
-- An intelligible name 
-- An entity among the list to which you want to associate the corresponding data
-- The format of the events
+7. Find your newly created source of events as well as the intake key in the Intakes homepage
 
-![image](https://user-images.githubusercontent.com/94441479/158421961-15b69898-a806-4d67-af0e-e52e2e359fc4.png)
+!!! note
+  
+     💡 The intake key able allows you to set up different elements such as Symphony modules.
```

Comment:
Not sure that we have a lot of Symphony modules that are using “intake keys”. 

We could replace that note with something explaining that one can use the intake key to search for events produced by a given system.